src/TokenRepository.php

```diff
@@ -27,6 +30,22 @@ public function save($token)
     }
 
     /**
+     * Store the given token instance.
+     *
+     * @param  Model  $userId
+     * @param  Client  $client
+     * @return Token|null
+     */
+    public function getValidToken($user, $client)
+    {
+        return $client->tokens()
+                      ->whereUserId($user->id)
+                      ->whereRevoked(0)
+                      ->where('expires_at', '>', Carbon::now()->toDateTimeString())
```

Comment:
I'm not sure you really need to make a string, you can just pass in the Carbon instance.